README.md

```
@@ -1,192 +1,172 @@
-  --left=shakapacker-config-exports/webpack-development-client.yaml \
-  --right=shakapacker-config-exports/webpack-production-client.yaml \
-  --format=summary
+pack-config-diff --left=webpack.dev.js --right=webpack.prod.js
 ```
 
-```bash
-# 2) Compare client vs server production bundles
-pack-config-diff \
-  --left=shakapacker-config-exports/webpack-production-client.yaml \
-  --right=shakapacker-config-exports/webpack-production-server.yaml
 ```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)